package.json

```diff
@@ -87,6 +87,16 @@
           }
         ],
         "priority": "default"
+      },
+      {
+        "viewType": "one.editor.jsonTracer",
+        "displayName": "chromeTrace",
+        "selector": [
+          {
+            "filenamePattern": "*.trace.json"
```

Comment:
One concern is the name could be anything (our backend profiling too have file name option from user)
Some possible solution could be
- Make constraint with file name: "File name must ends with `trace.json`. Maybe we need such constraint in `Profile` tab of cfg editor.
- Or can ONE explorer open json if json has, e.g., `"traceEvents"` key? (it seems chrome trace file has `"traceEvents"` key) /cc @dayo09

Reply:
I thought that `*.timeline.json` was used for profiling tool, but not sure :(

Reply:
> Or can ONE explorer open json if json has, e.g., "traceEvents" key? (it seems chrome trace file has "traceEvents" key) /cc @dayo09

Yes, it's possible. 

> I thought that *.timeline.json was used for profiling tool, but not sure :(

It's just a conventions, not a rule.

Reply:
I think that we all understand this issue now.
My opinion is... let's leave this line in this PR as it is and consider this issue in another PR. :-)

Reply:
@dayo I left a comment in the issue you mentioned. :-)